packages/reactive-dict/reactive-dict.js

```diff
@@ -114,7 +123,7 @@ _.extend(ReactiveDict.prototype, {
         Tracker.onInvalidate(function () {
           // clean up [key][serializedValue] if it's now empty, so we don't
           // use O(n) memory for n = values seen ever
-          if (! self.keyValueDeps[key][serializedValue].hasDependents())
+          if (!self.keyValueDeps[key][serializedValue].hasDependents())
```

Comment:
Please don't run a "prettifier" over the whole file when submitting a pull request, it results in a lot of unnecessary changes.